### DISCUSSION TITLE
How to spell check when typing in the git commit message box?

### DISCUSSION THREAD

**Discussion by ZYinMD** [streetsidesoftware/vscode-spell-checker#3531] [Q&A] [answered]
I know Code Spell Checker can show squiggles when I type in the git commit message box (shown in [this youtube video](https://www.youtube.com/watch?v=YjYtmFMkqmQ)), but it doesn't seem to work on my computer. When I test by typing a typo, nothing happens. Is there some additional setting I need? Thanks.

**Comment by Jason3S** [comment, marked as answer]
@ZYinMD,

Please try the `preview` version 4 of the Extension. It should just work. To get it to work in v3, you need to change the configuration a little bit by adding `vscode-scm` to the `cSpell.allowedSchemas` setting.
